### PR TITLE
fix(profile): Fix avatar not looking square

### PR DESCRIPTION
**Pull request by lutangar**

### CODE REVIEW

File: src/app/profiles/App/Pages/Profiles/Profile/ProfileIntro.tsx [hunk 1]
```diff
@@ -43,8 +43,8 @@ const ProfileIntroContent = styled.section`
 
         ${Avatar} {
           margin-top: 0;
-          width: 70px;
```
Comment (JalilArfaoui) [inline src/app/profiles/App/Pages/Profiles/Profile/ProfileIntro.tsx:46]: Do you know why this was malforming images ?

Comment (lutangar) [reply src/app/profiles/App/Pages/Profiles/Profile/ProfileIntro.tsx:46]: Not in details but border radius 50% and size restriction of the container were responsible.

**Comment by newick** [comment]
It comes from WordPress which has a 
```
img {
max-width: 100%;
}
```

if we reset `max-width: initial`, it works

**Review by JalilArfaoui — APPROVED** [review]

**Comment by lutangar** [comment]
Updated with @newick solution!

**Review by JalilArfaoui — APPROVED** [review]
💕

**Pull request merged**